setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="ubirch-protocol",
-    version="2.1.0",
+    version="3.1.0",
```

Comment:
version="3.1.0" has been released in the meantime

```suggestion
    version="3.2.0",
```